git-ftp

```diff
@@ -624,7 +624,7 @@ set_deployed_sha1() {
 	set_deployed_sha1_failable
 	local EXIT_CODE=$?
 	if [ "$EXIT_CODE" -eq 67 ]; then
-		print_error_and_die "Can't access remote 'ftp://test:***@127.0.0.1'. Failed to log in. Correct user and password? exiting..." "$ERROR_DOWNLOAD"
+		print_error_and_die "Can't access remote '$REMOTE_BASE_URL_DISPLAY'. Failed to log in. Correct user and password? exiting..." "$ERROR_DOWNLOAD"
```

Comment:
This is a detour in the commit history. In the first commit I saw this mistake and almost commented on it. Then I thought that you may fix it in a later commit and kept on reading, keeping this in mind. It's easier for reviewers if you remove these detours with `git rebase -i` before submitting the pull request. Here are some guidelines I like: https://github.com/openfoodfoundation/openfoodnetwork/wiki/Making-a-great-commit#commits-should-not-make-changes-that-are-effectively-reverted-in-a-subsequent-commit

Ideally, I like to read a pull request like a book, page by page, commit by commit, each of them making sense and passing the test suite.

Reply:
Thank’s for the tip. I used `git rebase -i` in other cases before but did not think about it in this case. I’ll try to remember in the future. Thanks.